readthedocs/projects/exceptions.py

```diff
@@ -1,15 +1,46 @@
 """Project exceptions"""
 
+from django.conf import settings
+from django.utils.translation import ugettext_noop as _
 
-class ProjectImportError (Exception):
+from readthedocs.doc_builder.exceptions import BuildEnvironmentError
 
-    """Failure to import a project from a repository."""
 
-    pass
+class ProjectConfigurationError(BuildEnvironmentError):
```

Comment:
It seems we have a `BuildEnvironmentException` and a `BuildEnvironmentError`, what is the difference? Seems confusing..

Reply:
Exception is the base class, Error is the specific class that kills a build and reports to a user, Warning also is absed on Exception and doesn't kill the build.